Source/WebCore/platform/graphics/nicosia/NicosiaBuffer.h

```diff
@@ -27,7 +27,7 @@
  */
 
 #pragma once
-
+#include "GraphicsContextGL.h"
```

Comment:
Can we forward declare the enum instead of including this header? or Use PixelFormat.h instead?

Reply:
Ok, PixelFormat.h can be chosen, as well. Eventually we need to extend it for ARGB.